VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/General/EqualsAndGetHashcodeNotImplementedTogether/EqualsAndGetHashcodeNotImplementedTogetherAnalyzer.cs

```diff
@@ -39,18 +45,17 @@ private void AnalyzeNode(SyntaxNodeAnalysisContext context)
                 }
 
                 var methodDeclaration = (MethodDeclarationSyntax)node;
-
                 if (!IsOverride(methodDeclaration))
                 {
                     continue;
                 }
 
-                if (methodDeclaration.Identifier.ValueText == nameof(Equals) && methodDeclaration.ParameterList.Parameters.Count == 1)
+                if (context.SemanticModel.GetDeclaredSymbol(methodDeclaration).OverriddenMethod == objectEquals)
```

Comment:
Of course, I could recurse the `OverriddenMethod` value to see if we ever reach the expected override on `Object`.  That should solve the problem.  To answer your question, `OverriddenMethod` refers to the method the current method is overridding.